tools/scripts-config-storybook-lib/.storybook-templates/preview.js

```diff
@@ -18,9 +18,13 @@ const i18n = initI18n(userI18n);
 let cmfLoader;
 let cmfDecorator;
 if (cmf) {
```

Comment:
Why is there is `if (cmf)` then 🤔

Reply:
to activate the import only if the user has done something. But compilation compile all branches of the code